### PR TITLE
Fix values rendering for Markdown engines with strict XML parsing

### PR DESCRIPTION
**Pull request by mindw**
* replace PWD with REPODIR in Makefile to fix run on PyCharm remote WSL2

### CODE REVIEW

File: hack/docs/template.tmpl [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {{- $notationType = "plaintext" }}
 {{- end -}}
 {{- end -}}
-<pre class="helm-vars-default-value" language-yaml" lang="{{ $notationType }}">
```
Comment (AndrewChubatiuk) [inline hack/docs/template.tmpl:10]: please revert this change everywhere as we're relying on this class in docs website

Comment (mindw) [reply hack/docs/template.tmpl:10]: @AndrewChubatiuk , Would `class="language-yaml"` do instead? what it now isn't a legal XML ...

Comment (AndrewChubatiuk) [reply hack/docs/template.tmpl:10]: why is this important? where do you use it?

**Comment by AndrewChubatiuk** [comment]
@mindw Thanks for contribution!

File: hack/docs/template.tmpl (hunk 1 above)
Comment (mindw) [reply hack/docs/template.tmpl:10]: My apologies - I wanted to suggest `helm-vars-default-value` - had a brain meltdown :( . 
Also, My CSS /XHTML is extremely hence missing options that there was an extra `"` .
